internal/guestpath/paths.go

```diff
@@ -0,0 +1,57 @@
+package guestpath
+
+import (
+	"path/filepath"
+	"runtime"
+)
+
+const (
+	// LCOWRootPrefixInUVM is the path inside UVM where LCOW container's root file system will be mounted
+	LCOWRootPrefixInUVM = "/run/gcs/c"
+	// WCOWRootPrefixInUVM is the path inside UVM where WCOW container's root file system will be mounted
+	WCOWRootPrefixInUVM = `C:\c`
+	// SandboxMountPrefix is mount prefix used in container spec to mark a sandbox-mount
+	SandboxMountPrefix = "sandbox://"
+	// HugePagesMountPrefix is mount prefix used in container spec to mark a huge-pages mount
+	HugePagesMountPrefix = "hugepages://"
+	// SandboxMountsPathWCOW is the path for WCOW sandbox mounts.
+	SandboxMountsPathWCOW = "C:\\SandboxMounts"
+)
+
+// NetworkingMountPaths returns an array of mount paths to enable networking
+// inside containers.
+func NetworkingMountPaths() []string {
+	if runtime.GOOS != "linux" {
+		return nil
+	}
+
+	return []string{
+		"/etc/hostname",
+		"/etc/hosts",
+		"/etc/resolv.conf",
+	}
+}
+
+// SandboxRootDir returns the sandbox container root directory inside UVM/host.
+func SandboxRootDir(sandboxID string) string {
+	if runtime.GOOS == "linux" {
+		return filepath.Join(LCOWRootPrefixInUVM, sandboxID)
+	}
+	return filepath.Join(WCOWRootPrefixInUVM, sandboxID)
+}
+
+// SandboxMountsDir returns sandbox mounts directory inside UVM/host.
+func SandboxMountsDir(sandboxID string) string {
+	if runtime.GOOS == "linux" {
```

Comment:
Can this just be Linux specific? AFAIK we don't actually use this for Windows anywhere, since the gcs is Linux only.

Reply:
done